### PR TITLE
improve help for api command

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

**Review by mattnibs — APPROVED** [review]

File: cmd/zed/api/command.go [hunk 1]
```diff
@@ -15,8 +15,17 @@ import (
...
+	Short: "perform lake actions on Zed service",
+	Long: `
+The "api" commands provides client access to a Zed lake service running
+on the IP and port provided in the "-host" option.  This option defaults
+to localhost:9867 so you can conveniently connect to a lake service
+running locally on the default port, as is automatically launched
+by the Brim application for the "local Zed lake".
+
+All of the relevant "lake" commands are available through the "api" command.
+Refer to the help of the indidual sub-commands for more details.`,
```
Comment (philrz) [inline cmd/zed/api/command.go:27]:
```suggestion
Refer to the help of the individual sub-commands for more details.`,
```

File: cmd/zed/api/command.go [hunk 2]
```diff
@@ -15,8 +15,17 @@ import (
 var Cmd = &charm.Spec{
 	Name:  "api",
 	Usage: "api [options] sub-command",
-	Short: "create, manage, and search Zed lakes",
-	New:   New,
+	Short: "perform lake actions on Zed service",
+	Long: `
+The "api" commands provides client access to a Zed lake service running
```
Comment (nwt) [inline cmd/zed/api/command.go:20]:
```suggestion
The "api" commands provide client access to a Zed lake service running
```

Comment (philrz) [inline cmd/zed/api/command.go:20]:
```suggestion
The "api" commands provide client access to a Zed lake service running
```

**Review by philrz — APPROVED** [review]
Found a couple typos, but overall LGTM. 👍

**Pull request merged**